### DISCUSSION TITLE
Need assistance - page text not returning properly

### DISCUSSION THREAD

**Discussion by Smilingval62** [microsoft-search/pnp-modern-search#2625] [Q&A]
We have installed PnP in our SharePoint Online tenant. 
We have a page with multiple text webparts and we would like to return the text when users search.
Currently when we search we are only getting the page itself returned and "found" text isn't highlighted - in addition we would like the section to highlight not just the page. See attached file for examples. 
**Version used**
Ex: 4.8

**Describe the bug**
A clear and concise description of what the bug is. Keep questions and bugs related to the web parts. If the issue relates to KQL or question around crawled and managed properties please use [Microsoft TechCommunity](https://techcommunity.microsoft.com/t5/sharepoint-developer/bd-p/SharePointDev). Which means, if you experience the same behavior in classic web parts, then it's not an issue with PnP Modern Search 😉

**To Reproduce**
Detailed steps to reproduce the behavior:
4. Enter setting A, B, C
5. See error

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**
[SearchResults.pdf](https://github.com/microsoft-search/pnp-modern-search/files/10127162/SearchResults.pdf)

**Comment by Smilingval62** [comment]
Edge is the browser we are using

**Comment by wobba** [comment]
Not clear what you are asking. But if you want to show metadata from the results, then you have to get that data in a managed property you can use in a customized layout.
